{"uuid": "008bf288-49c5-44a1-b602-bf813f60f479", "children": ["2d14b6ba-906b-4a99-9cd3-574164b96f4c", "cc9ce4d6-3c99-4173-895c-6cafa7cec965"], "befores": [{"name": "test_setup", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 81\n\n", "trace": "  File \"C:\\Users\\Vignesh\\PycharmProjects\\AutomationFramework1\\venv\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\Vignesh\\PycharmProjects\\AutomationFramework1\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\Vignesh\\PycharmProjects\\AutomationFramework1\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 788, in call_fixture_func\n    res = next(it)\n  File \"C:\\Users\\Vignesh\\PycharmProjects\\AutomationFramework1\\conftest.py\", line 6, in test_setup\n    driver = webdriver.Chrome(\"C://Users//Vignesh//PycharmProjects//Selenium//driver//chromedriver.exe\")\n  File \"C:\\Users\\Vignesh\\PycharmProjects\\AutomationFramework1\\venv\\lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py\", line 76, in __init__\n    RemoteWebDriver.__init__(\n  File \"C:\\Users\\Vignesh\\PycharmProjects\\AutomationFramework1\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 157, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"C:\\Users\\Vignesh\\PycharmProjects\\AutomationFramework1\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 252, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n  File \"C:\\Users\\Vignesh\\PycharmProjects\\AutomationFramework1\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\Vignesh\\PycharmProjects\\AutomationFramework1\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1590779443886, "stop": 1590779449209}], "start": 1590779443886, "stop": 1590779450343}